src/main/java/io/stargate/sgv2/jsonapi/service/embedding/configuration/EmbeddingProviderConfigStore.java

```diff
@@ -31,15 +31,22 @@ record RequestProperties(
       int retryDelayInMillis,
       int timeoutInMillis,
       Optional<String> requestTypeQuery,
-      Optional<String> requestTypeIndex) {
+      Optional<String> requestTypeIndex,
+      int batchSize) {
     public static RequestProperties of(
         int maxRetries,
         int retryDelayInMillis,
         int timeoutInMillis,
         Optional<String> requestTypeQuery,
-        Optional<String> requestTypeIndex) {
+        Optional<String> requestTypeIndex,
+        int batchSize) {
```

Comment:
Should this be named "maxBatchSize" where it defines maximum batch provider allows?